crates/warpgate/src/oci_registry.rs

```diff
@@ -0,0 +1,8 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize, schematic::Schematic)]
+#[serde(default, rename_all = "kebab-case")]
+pub struct OciRegistry {
+    pub registry: String,
+    pub organization: Option<String>,
```

Comment:
The image name?

A lot of this naming is confusing lol.

Reply:
;) for the configuration you do not need the image name, registry and organization is sufficient.

Exmaple
```
Registry     = docker.io      The host
Organization = moonrepo       Subfolder/Namespaces from your requirements
Image        = java           The container/image which has the WASM as layer included
```